ee/codegen/src/generators/node-inputs/node-input-value-pointer-rules/execution-counter-pointer.ts

```diff
@@ -12,12 +12,10 @@ export class ExecutionCounterPointerRule extends BaseNodeInputValuePointerRule<E
       executionCounterData.nodeId
     );
 
-    const inputRef = python.reference({
+    return python.reference({
       name: nodeContext.nodeClassName,
       modulePath: nodeContext.nodeModulePath,
       attribute: ["Execution", "counter"],
     });
-    this.addReference(inputRef);
-    return inputRef;
```

Comment:
We no longer have to manually add references because the base class does this with the return value of this function.